extensions/mongoProcessor/MongoQueueProcessor.js

```diff
@@ -97,32 +77,15 @@ class MongoQueueProcessor {
      * @return {undefined}
      */
     start() {
-        this.logger.info('starting mongo queue processor');
-        async.series([
-            next => this._setupMetricsClients(err => {
-                if (err) {
-                    this.logger.error('error setting up metrics client', {
-                        method: 'MongoQueueProcessor.start',
-                        error: err,
-                    });
-                }
-                return next(err);
-            }),
-            next => this._mongoClient.setup(err => {
-                if (err) {
-                    this.logger.error('could not connect to MongoDB', {
-                        method: 'MongoQueueProcessor.start',
-                        error: err.message,
-                    });
-                }
-                return next(err);
-            }),
-        ], error => {
-            if (error) {
+        this._mongoClient.setup(err => {
+            if (err) {
+                this.logger.error('could not connect to MongoDB', {
+                    method: 'MongoQueueProcessor.start',
+                    error: err.message,
+                });
                 this.logger.fatal('error starting mongo queue processor');
```

Comment:
No need to log twice I would say, just log the fatal message with all the info

```suggestion
                this.logger.fatal('could not connect to MongoDB, error starting mongo queue processor', {
                    method: 'MongoQueueProcessor.start',
                    error: err.message,
                });
```